inc/saiport.h

```diff
@@ -232,6 +232,8 @@ typedef enum _sai_port_priority_flow_control_mode_t
     /** Separate values for RX/TX */
     SAI_PORT_PRIORITY_FLOW_CONTROL_MODE_SEPARATE,
 
+    /** Disable PFC for both rx and tx */
+    SAI_PORT_PRIORITY_FLOW_CONTROL_MODE_DISABLED,
```

Comment:
Is this attribute necessary?  Without this, PFC could still be disabled using SAI_PORT_ATTR_PRIORITY_FLOW_CONTROL or (SAI_PORT_ATTR_PRIORITY_FLOW_CONTROL_RX and SAI_PORT_ATTR_PRIORITY_FLOW_CONTROL_TX).

Reply:
the problem here is for gearbox, they do not control per bitmap. they can do disable for all priority or none. the per bit control is not supported on the gearbox.